db/migrate/20220603195131_create_users.rb

```diff
@@ -0,0 +1,12 @@
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.text :photo
```

Comment:
- [ ] kindly change this datatype to string because it's more likely to hold a url to an image

Reply:
Ok, thank you.